src/client/kotlin/me/owdding/skyocean/features/inventory/buttons/InvButtons.kt

```diff
@@ -0,0 +1,100 @@
+import tech.thatgravyboat.skyblockapi.helpers.McClient
+import tech.thatgravyboat.skyblockapi.utils.extentions.containerHeight
+import tech.thatgravyboat.skyblockapi.utils.extentions.containerWidth
+import tech.thatgravyboat.skyblockapi.utils.extentions.left
+import tech.thatgravyboat.skyblockapi.utils.extentions.top
+import tech.thatgravyboat.skyblockapi.utils.text.Text
+
+@Module
+object InvButtons {
+    @Subscription
+    fun onScreen(event: ScreenInitializedEvent) {
+        if (showButtons(event.screen)) return
+        val screen = event.screen as AbstractContainerScreen<*>
+
+        val buttonOffset = 8
+        val buttonHeight = 32
+
+        for (y in 0 until 2) {
+            for (x in 0 until 7) {
+                val posX = (x * (screen.containerWidth / 7)) + screen.left
+                val yOffset = if (y == 0) {
+                    -buttonHeight + buttonOffset
+                } else {
+                    -buttonOffset - 1
+                }
+                val posY = (y * screen.containerHeight) + screen.top + yOffset
+
+                val button = Buttons.buttons[x + y * 7]
+                event.widgets.add(
+                    InvButton(button, x, y == 1, screen, x + y * 7, posX, posY, 26, buttonHeight)
+                        .withSize(26, buttonHeight)
+                        .withCallback {
+                            if (screen is ButtonConfigScreen) {
+                                screen.refresh(x + y * 7)
+                            } else {
+                                val command = button.command.replace("/", "")
+                                McClient.connection?.sendCommand(command)
+                            }
+                        }
+                        .withTooltip(Text.of(button.tooltip.takeIf { it.isNotEmpty() } ?: button.command))
+                        .withPosition(posX, posY)
+                )
+            }
+        }
+    }
+
+    @Subscription
+    fun onScreenBackground(event: RenderScreenBackgroundEvent) {
+        if (showButtons(event.screen)) return
+        Screens.getButtons(event.screen).forEach {
+            if (it is InvButton && !it.highlight) {
+                it.renderButtons(event.graphics, 0, 0, 0F)
+            }
+        }
+    }
+
+    @Subscription
+    fun onScreenForeground(event: RenderScreenForegroundEvent) {
+        if (showButtons(event.screen)) return
+        Screens.getButtons(event.screen).forEach {
+            if (it is InvButton && it.highlight) {
+                it.renderButtons(event.graphics, 0, 0, 0F)
+            }
+        }
+    }
+
+    @Subscription
+    fun onCommand(event: RegisterSkyOceanCommandEvent) {
+        event.registerWithCallback("buttons") {
+            if (InventoryConfig.inventoryButtons) {
+                McClient.setScreenAsync(ButtonConfigScreen(null))
+            } else {
+                Text.of("First Enable Inventory Buttons in the Config").withColor(0xf38ba8).sendWithPrefix()
```

Comment:
missing translations & also
```suggestion
                Text.of("Enable inventory buttons in the Config first!").withColor(0xf38ba8).sendWithPrefix()
```